adapters/handlers/grpc/v1/parse_search_request.go

```diff
@@ -364,46 +365,104 @@ func extractGroupBy(groupIn *pb.GroupBy, out *dto.GetParams) (*searchparams.Grou
 	return groupOut, nil
 }
 
-func extractTargetVectors(req *pb.SearchRequest, class *models.Class) (*[]string, error) {
+func extractTargetVectors(req *pb.SearchRequest, class *models.Class) (*[]string, *dto.TargetCombination, error) {
 	var targetVectors *[]string
+	var targets *pb.Targets
+
+	extract := func(targets *pb.Targets, targetVectors *[]string) (*[]string, *pb.Targets) {
```

Comment:
can we use just return a slice `[]string` instead of a pointer to slice `*[]string`? that would then simplify all of the readability of the code and all our checks.

Reply:
bc now we need to do checks like, which I think is unnecessary:
```
if targetVectors != nil && len(*targetVectors) > 1
```
it would change to:
```
if len(targetVectors) > 1
```

Reply:
yep, changed :)